examples/bri-3/src/bri/zeroKnowledgeProof/api/anchorHash.controller.ts

```diff
@@ -15,7 +15,11 @@ export class AnchorHashController {
     @Body() requestDto: CreateAnchorHashDto,
   ): Promise<AnchorHashDto> {
     return await this.commandBus.execute(
-      new CreateAnchorHashCommand(requestDto.ownerAccount, requestDto.state),
+      new CreateAnchorHashCommand(
```

Comment:
Why is there a need for an endpoint that creates an anchor hash from a state submitted from the outside? Shouldn't the anchor hash creation and storage happen as a consequence of a state change, after a workstep has been successfully executed?

Reply:
@ognjenkurtic I agree that your logic seems correct. Can anyone provide a reason? @biscuitdey @Therecanbeonlyone1969 ? Otherwise this is existing code, are we ok with me getting rid of it ?